src/integration-test/java/org/openlmis/stockmanagement/web/ValidSourceDestinationControllerIntegrationTest.java

```diff
@@ -87,6 +87,31 @@ public void shouldGetValidSourcesOrDestinationsByProgramAndFacility()
     performSourcesOrDestinations(program, facility, sourceDestination, API_VALID_SOURCES);
   }
 
+  @Test
+  public void shouldGeAllValidSourcesOrDestinationsWhenProgramAndFacilityAreNotProvided()
+          throws Exception {
```

Comment:
In this integration test, we are using `MockMvc` which throws Exception. We must leave `throws Exception` because without it the code cannot be compiled.